index.html

```diff
@@ -110,6 +146,56 @@
   </p>
 </section>
 
+<!-- - - - - - - - - - - - - - -  Introduction  - - - - - - - - - - - - - - -->
+<section class="informative"> <h2>Introduction</h2>
+  <p>
+    In general, there are following groups of user scenarios for NFC:
+    <ul>
+      <li>
+        Hold a device in close proximity to a passively powered tag, such as
+        a plastic card or sticker, in order to read and/or write data.
+      </li>
+      <li>
+        Hold two active devices, e.g. phones or tablets, in close proximity
+        in order to push an <a>NDEF message</a> from one device to the other.
```

Comment:
why say NDEF message here and 'data' above? Maybe we should just say data

Reply:
Fixed